docs/sdk/typescript.mdx

```diff
@@ -10,31 +10,28 @@ Learn how to use the Mogami TypeScript SDK
 # Installation
 
 ```shell
-yarn add @mogami/sdk \
-         @mogami/keypair \
-         @mogami/solana \
-         @solana/web3.js \
-         @solana/spl-token \
-         axios \
-         bip39 \
-         ed25519-hd-key \
-         tslib
+yarn add @mogami/sdk @mogami/keypair @mogami/solana 
 ```
 
 ## Initialize SDK
 
+Initialise the SDK so that you have access to all of Mogami's methods.
```

Comment:
I think here we want to have a note that tells people to configure the `endpoint` property one a self-hosted instance